activestorage/test/models/attachments_test.rb

```diff
@@ -68,7 +68,7 @@ class ActiveStorage::AttachmentsTest < ActiveSupport::TestCase
   test "attach new blobs" do
     @user.highlights.attach(
       { io: StringIO.new("STUFF"), filename: "town.jpg", content_type: "image/jpg" },
-      { io: StringIO.new("IT"), filename: "country.jpg", content_type: "image/jpg" })
+      io: StringIO.new("IT"), filename: "country.jpg", content_type: "image/jpg")
```

Comment:
Can we add a rubocop comment disabling this role in those places. We discussed before and I agree for this case it is weird to have one with `{}` and others without. So we decided to keep both hashes consistent even that it goes against the rubocop rule